sources/packages/web/src/views/student/application-offering-change-request/tabs/RequestedApplicationOfferingDetails.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <tab-container :enableCardView="false" class="mt-n1">
+    <body-header-container>
+      <template #header>
+        <body-header :title="studentFullName">
+          <template #subtitle
+            ><detail-header :headerMap="headerDetailsData"
+          /></template>
+          <template #status-chip>
+            <status-chip-application-offering-change
+              :status="changeRequest.status"
+            />
+          </template>
+        </body-header>
+      </template>
+      <hr class="horizontal-divider" />
+      <h2 class="category-header-large primary-color">Application Details</h2>
+      <p v-if="changeRequestNotApproved">
+        Below displays the requested changes from your institution. You can
+        compare your <b>active application details</b> by switching tabs located
+        above.
+      </p>
+      <offering-view :offeringId="changeRequest.requestedOfferingId" />
+      <hr class="horizontal-divider" />
+      <h2 class="category-header-large primary-color">Request details</h2>
+      <p v-if="changeRequestNotApproved">
+        Contact your institution for more information if there are no
+        differences in the requested change above. There may be proposed changes
+        that aren't displayed such as study costs.
+      </p>
+      <content-group-info
+        ><div class="mb-2">
+          <span class="category-header-medium primary-color"
```

Comment:
this is an h3 header, 
![image](https://github.com/bcgov/SIMS/assets/77353155/bebc40be-2e2f-448c-ae6f-fd68071282c0)